src/components/navBar/ NavBar.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from "react";
+import {useState}  from 'react';
+import "./navBar.css";
+// import Burger from "../Burger/burger"
+const NavBar = () => {
+
+  const [showLinks,setShowLinks] = useState<boolean>(true)
+
+  return (
+    <section className="navbar-container">
+      <img onClick ={() => setShowLinks(!showLinks)} src="../images/group-13.png" className="burger-icon" alt="burger-icon"></img>
+
+    {/* <Burger /> */}
+      <div className="left-nav">
+        <img src="../images/logo.png" className="logo-icon" alt="logo-icon"></img>
+        <div className="logo">EPICURE</div>
+        <a href="/" className="menu-link">Restaurants</a>
+        <a href="/" className="menu-link">Chefs</a>
+      </div>
+
+      <div className="right-nav">
+        <div className="search-container">
+          <input type="text" placeholder="Search for restaurant cuisne, chef" className="search"></input>
+          <img src="../images/search-icon.png" className="search-icon" alt="search-icon"></img>
```

Comment:
When writing input or img tags theres no need to use a closing tag, you can use a self closing tag instead:
<img src="koko.png" alt="koko akof" />

Reply:
Thanks! I have an extension that auto-complete html tags.
Maybe I could disable it for this tags.